ProcessLib/DeactivatedSubdomain.cpp

```diff
@@ -89,49 +89,39 @@ std::unique_ptr<DeactivatedSubdomain const> createDeactivatedSubdomain(
 
     for (auto const ids : deactivated_subdomain_material_ids)
     {
-        std::vector<MeshLib::Element*> deactivated_elements;
+        auto const& nodes = mesh.getNodes();
         std::vector<MeshLib::Node*> deactivated_nodes;
+        for (auto const& node : nodes)
+        {
+            const auto& connected_elements = node->getElements();
+
+            // Check whether this node is in an activated element.
+            if (std::find_if(
+                    connected_elements.begin(),
+                    connected_elements.end(),
+                    [&](auto const* const connected_elem) -> bool {
+                        return ids != (*material_ids)[connected_elem->getID()];
+                    }) != connected_elements.end())
+            {
+                continue;
+            }
+            else
```

Comment:
no need for an `else` branch after `return`, `continue`, `break`...

Reply:
Forgot to remove that after removed lines except for `continue;` in if block. Removed the else by amending the commit.